### ISSUE TITLE
unwanted line of code added after_TradeMacroMain.ahk

### ISSUE THREAD

**Issue by JustTheNobody** [PoE-TradeMacro/POE-TradeMacro#928]
Hi all, 

I have error in _TradeMacroMain.ahk after I have been creating text file for some another project. that file is linked to the script for some reason and even if I delete the line in question after next run it's added back to the file. I have reinstall the trade macro and the AHK it self to but the problem remind same. I have not find any solution to this by using google search so I'm writing in hope that someone know what to do.

I did little video for my friend in hope that he will know more about that so you can check here: https://www.youtube.com/watch?v=j1Mp0BdXiEA&feature=youtu.be&fbclid=IwAR1y3glip4pB8-P4qq2ONntZXILnElx3_3OrRXludsIZBbmQ_vmwwpTpfWY

AHK version:  v1.1.30.01
Windows version: windows 10 (64bit)

thanks guys

**Comment by Eruyome** [comment]
- Delete the file `...\Documents\PoE-TradeMacro\CustomMacros\CZ todo.txt`, it will always be added again if it exists in this CustomMacros folder.

**Comment by JustTheNobody** [comment]
Hi,
Thanks for your reply, I have deleted it and than I did another search for a file with same name to be on safe side and discover I have like 3 copies of the macro with the files in it. I deleted all and it's all good now.
Thanks a lot!!!!
with regards
Martin Malymartin.maly_77@yahoo.com 

    On Sunday, 3 March 2019, 12:07:04 GMT, Eruyome <notifications@github.com> wrote:  
 
    
   - Delete the file ...\Documents\PoE-TradeMacro\CustomMacros\CZ todo.txt, it will always be added again if it exists in this CustomMacros folder.
